templates/dashboards/grafana-dashboard-image-builder-worker-general.configmap.yml

```diff
@@ -2555,6 +2673,47 @@ data:
             "query": "0.95",
             "skipUrlSync": false,
             "type": "constant"
+          },
+          {
+            "current": {
```

Comment:
hey @chloenayon sorry for the change request on this :/
See this conversation here:
https://github.com/RedHatInsights/image-builder-frontend/pull/753#discussion_r921235240

Could you please update this to have an "All" option and making it the default. I think you can add it through the GUI:
![image](https://user-images.githubusercontent.com/20438192/179025969-f3414c11-dfbc-49fa-87a2-b7ca31088092.png)

Reply:
Setting the custom all value to `.*` will give us a regex wildcard